payment-request/blank.html

```diff
@@ -1 +1,14 @@
-<!DOCTYPE html> <meta charset="utf-8" />
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Blank Page</title>
+    <script>
+      window.onload = function(event) {
+        console.log("Loaded ", window.location.href);
```

Comment:
```suggestion
```

Reply:
We can leave the no op, but no need for the console.log()

Reply:
Done. I added a comment to document why this empty onload function is necessary.